### DISCUSSION TITLE
NodeJs Version

### DISCUSSION THREAD

**Discussion by castasint** [microsoft/FHIR-Converter#237] [Q&A: Handlebars Engine]
Hi - I see that there is a constraint on the NodeJs version when using ` "engines": {
    "node": ">=10.10.0 <11.0"
  },`. 30th of this month(04/30/2021) is the end-of-life for any 10.x versions. So, is there any plan of upgrading the project to v14 or v12(LTS) so it can be safely used? Please advise.

Also, if there is a guide that already explains how the project can be upgraded, please point me towards it. Happy to answer any further questions.

Thanks,
Hanuma

![image](https://user-images.githubusercontent.com/16588875/116384098-260c3780-a835-11eb-9cd8-039f29314276.png)

**Comment by ginalee-dotcom** [comment]
Implemented in https://github.com/microsoft/FHIR-Converter/pull/265.

**Comment by yaxionglin** [comment]
Could you share the Dockerfile that builds from the latest FHIR-Converter (5.1.0)?

Below is my Dockerfile, it builds container ok but it is throwing the error when running the container: "no such file or directory, open '/app/package.json'"

FROM node:14 AS build
ARG version=v5.1.0
RUN git clone --depth=1 --branch ${version} https://github.com/microsoft/FHIR-Converter.git /app
WORKDIR /app
RUN npm install --only=production --no-fund --no-optional --no-audit
EXPOSE 2019
CMD [ "npm", "start" ]
